paper.tex

```diff
@@ -627,11 +627,11 @@ \subsection{GPU: Execution Time} \label{subsection:gpu-results}
     \label{table:bump-counts}
 \end{table}
 
-The test scenes were rendered to a GPU texture with dimensions 2088x1600. The contents of each scene were uniformly scaled up (by encoding a transform) to fit the size of the texture, in order to increase the required number of output primitives. As with the CPU timings, we used an error tolerance of 0.25 in all cases. We recorded the execution times across 3,000 GPU submissions for each scene.
+The test scenes were rendered to a GPU texture with dimensions 2088x1600. The contents of each scene were uniformly scaled up (by encoding a transform) to fit the size of the texture, in order to increase the required number of output primitives. As with the CPU timings, we used an error tolerance of 0.25 pixels in all cases. We recorded the execution times across 3,000 GPU submissions for each scene.
 
 \begin{figure}
     \includegraphics[scale=0.69]{nehab_gpu_timings}
-    \caption{Mean GPU execution times of the compute shader that performs stroke expansion and flattening when run on the \citet{Nehab2020} \emph{timings} data set. The timings are shown in milliseconds. The hatched bars show the timings for the version that outputs line primitives while the solid bars show that for arcs. Mali-G78 (the lowest end GPU we tested) converts \emph{waves} (the largest scene in the set) to lines in ~3ms. The compute shader executes at least an order of magnitude faster on the other GPUs.}
+    \caption{Mean GPU execution times of the compute shader that performs stroke expansion and flattening when run on the \citet{Nehab2020} \emph{timings} data set. The timings are shown in milliseconds. The hatched bars show the timings for the version that outputs line primitives while the solid bars show that for arcs. Mali-G78 (the lowest-end GPU we tested) converts \emph{waves} (the largest scene in the set) to lines in \~3~ms. The compute shader executes at least an order of magnitude faster on the other GPUs.}
```

Comment:
> \~3~ms

Is this right? On the printout this appears with a ~ above the 3 like this:
<img width="93" alt="Screenshot 2024-06-13 at 5 00 16 PM" src="https://github.com/linebender/gpu-stroke-expansion-paper/assets/6933700/212b0a89-3e46-4d73-a441-b0baa7777caa">

Reply:
Maybe the English word _about_ is a better fit.